.github/workflows/build.yml

```diff
@@ -251,3 +251,10 @@ jobs:
       - name: Exit
         shell: bash
         run: exit 0
+
+  build_iso:
+    name: build iso
+    needs: [check]
+    if: github.event_name == 'pull_request' || github.ref_name == 'testing'
```

Comment:
A few thoughts on this condition:
1. using OR here means all pull requests will try to build an ISO, maybe fine now, but I was originally thinking of `pull_request AND 'testing'`, though I'm not sure it would work
2. i think you may also want       `  if: needs.push-ghcr.result == 'success'` or at least to add that as another `&&`  so the container build won't try to build an ISO unless it was a clean success

Reply:
Building an ISO on all pull requests would be ideal in my opinion. It means every single PR has to be able to successfully create a container and ISO in order to be merged. I don't know if an `AND` would work here.

Reply:
The vast majority of changes have no bearing on ISO building success. And its a long, slow process which eats our builder time. So every time Jorge switches a wallpaper or adds a font, we'd have to build ISOs. I personally don't see the need, which probably gets back to some of the discussion we had earlier. :smile: 

Regardless, even my suggestion of `PR && testing` may not help my argument much if changes are typically made to the testing branch.

Reply:
Yeah. Maybe we can have a switch or something in the workflow to turn on building ISOs if that is something needed during testing?